signature/Cargo.toml

```diff
@@ -36,15 +36,16 @@ solana-short-vec = { workspace = true }
 solana-signature = { path = ".", features = ["serde"] }
 
 [features]
-default = ["std"]
+default = ["std", "alloc"]
```

Comment:
Since "std" implies "alloc", is it needed to specify both here?

Reply:
I was thinking to eventually remove the `std` feature in favor of `alloc`, and by having it already set as a default it'll be simpler (and maybe less breaking?) to make that change in the future.